Windows/jenkins-node/readme.md

```diff
@@ -102,3 +102,56 @@ Server: Mirantis Container Runtime
 9. Finally, change the label on the node back to `win-64-cloud` via `Configure` (and reset the preference score if applicable). This will enable the node to take part in the CI/CD pipeline.
+
+# Cloud Windows Jenkins Node Troubleshooting
+
+## First response to failure
+
+If a cloud windows Jenkins node has run into issues, the first port of call should be to stop, remove, and rerun the docker container:
+
+### Bring the node offline
+
+1. Log in to the `Jenkins` web UI (https://builds.mantidproject.org/), navigate to `Manage Jenkins`, then `Manage nodes and clouds` under `System Configuration`.
+2. On the `Manage nodes and clouds` page of Jenkins, select the node in question from the table.
+3. Ensure the node is offline if it isn't already: If `Mark this node temporarily offline` is available on the right of screen, select it.
+
+### Restart the container and bring back online
+
+1. On your local machine, [connect to the VM hosting the node in question using the instructions outlined](##-connect-to-the-windows-cloud-virtual-machine).
+2. Open `powershell` in administrator mode.
+3. Stop the container: `docker stop <cloud node name>`.
+4. Remove the container: `docker rm <cloud node name>`.
+5. Run the container: 
+   ```sh
+   docker run -d --name <cloud node name> --storage-opt "size=250GB" --restart on-failure:3 <docker image name> -Url https://builds.mantidproject.org -Secret <jenkins secret> -WorkDir C:/jenkins_workdir -Name <cloud node name>
+   ```
+
+6. Confirm that the container has been created and is listed as running using `docker container ps -a`.
+7. On the `Jenkins` web UI, refresh the page and ensure that `Agent is connected` is displayed. Click `Bring this node back online`.
+
+## Upon continued failure
+
+If a cloud windows Jenkins node continues to fail having been rerun, the issue may be with the VM which is hosting it. To restart this VM:
+
+1. Bring the node offline as detailed in the above [section](###-bring-the-node-offline).
+2. On your local machine, [connect to the VM hosting the node in question](##-connect-to-the-windows-cloud-virtual-machine).
+3. Using the start menu, simply click the power symbol and select `Restart`.
+4. After a short period, [restart the container and bring it back online](###-restart-the-container-and-bring-back-online)
+
+## Upon failure to restart
+
+If the virtual machine fails to come back online after restart, or if issues persist after restart, the VM should be restarted via the `Virtual Machine Manager Console` (VMMC):
+
+### Install the VMMC
+
+1. Access `\\FITCLOUDVMMSB2\VMM2016ConsoleInstall$`. If you cannot access this file location, you will have to be granted permissions: please contact a member of the RAL Mantid Devops team.
+2. In the subdirectory `BaseInstallVMM2016` run `setup.exe`. Select `Install`, then all defaults as you progress through the installation.
```

Comment:
I need to tick the `VMM Console` feature to make sure it was installed (It was not ticked by default)